.circleci/config.yml

```diff
@@ -76,12 +76,13 @@ jobs:
           command: |
             if [[ "$CIRCLE_PULL_REQUEST" == "" && "$CIRCLE_BRANCH" == "develop" ]] ; then
                 docker-compose exec web yarn build
-                if [[ `git status static/dist/ webpack-stats.json --porcelain` ]] ; then
-                    docker-compose exec web fab update_docker_image_version
+                docker-compose exec web fab update_docker_image_version
+                if [[ `git status static/dist/ webpack-stats.json docker-compose.yml --porcelain` ]] ; then
                     git config user.email "lil@law.harvard.edu"
                     git config user.name "Circle CI"
                     git add static/dist/ webpack-stats.json docker-compose.yml
-                    git commit -m 'Add built JS [skip ci]'
+                    JSCHANGED=`git diff --name-only --cached | grep -q stat && echo ', add built JS'`
```

Comment:
If you want the commit message to be precise, note that both are optional -- `git status static/dist/ webpack-stats.json` implies JS was rebuilt, and `git status docker-compose.yml` implies docker version was updated.

Reply:
You could also break them out entirely as yarn build, commit changes to static/dist/ webpack-stats.json if any; fab update_docker_image_version, commit changes to docker-compose.yml if any.

Reply:
Ah, right. I was thinking that if we were in this stanza, we'd by definition have a new Docker image number, but that's wrong. Hold one, I'll push a fix.